### PR TITLE
Master services are gone in 3.10

### PR DESCRIPTION
**Pull request by vrutkovs**
There's nothing to stop or reload in 3.11

### CODE REVIEW

File: playbooks/common/openshift-cluster/upgrades/docker/tasks/upgrade.yml [hunk 1]
```diff
@@ -2,13 +2,9 @@
...
 - name: Stop containerized services
-  service: name={{ item }} state=stopped
-  with_items:
-  - "{{ openshift_service_type }}-master-api"
-  - "{{ openshift_service_type }}-master-controllers"
-  - "{{ openshift_service_type }}-node"
-  - etcd_container
-  - openvswitch
+  service:
+    name: etcd_container
```
[the remark this answers is not in the document]
Comment (michaelgugino) [reply playbooks/common/openshift-cluster/upgrades/docker/tasks/upgrade.yml]: It's not.  containerized nothing is supported, only rpm standalone or static pods.

[the remark this answers is not in the document]
Comment (michaelgugino) [reply playbooks/common/openshift-cluster/upgrades/docker/tasks/upgrade.yml]: No, we can get rid of this task completely

[the remark this answers is not in the document]
Comment (vrutkovs) [reply playbooks/common/openshift-cluster/upgrades/docker/tasks/upgrade.yml]: Done

File: playbooks/common/openshift-cluster/upgrades/docker/tasks/upgrade.yml [hunk 2]
```diff
@@ -1,17 +1,6 @@
 ---
 # We need docker service up to remove all the images, but these services will keep
 # trying to re-start and thus re-pull the images we're trying to delete.
-- name: Stop containerized services
-  service: name={{ item }} state=stopped
-  with_items:
-  - "{{ openshift_service_type }}-master-api"
-  - "{{ openshift_service_type }}-master-controllers"
-  - "{{ openshift_service_type }}-node"
```
Comment (vrutkovs) [inline playbooks/common/openshift-cluster/upgrades/docker/tasks/upgrade.yml]: This might break atomic, would be re-introduced in #9674

**Comment by vrutkovs** [comment]
scandir fix has been merged
/test gcp

**Pull request merged**